versions/1.0/parsers/antlr4/WdlParser.g4

```diff
@@ -157,21 +158,25 @@ struct
 	: STRUCT Identifier LBRACE (unbound_decls)* RBRACE
 	;
 
-meta_kv
-	: Identifier COLON expr
+meta_expr
+	: expr
+	| Null
```

Comment:
Null is not a valid symboll in V1. Additionally, in a future version of the parser its `None` and its placed within the expression resolution, howver you would not explicitly have to add it there, since it it added to the `primitive_literal` rule

Reply:
I added it to primitive_literal by mistake. But according to the 1.0 spec it is a valid value within the meta sections.